protected/humhub/components/console/Application.php

```diff
@@ -131,4 +132,11 @@ public function setHomeUrl($value)
         $this->_homeUrl = $value;
     }
 
+    /**
+     * @return Session the session component.
+     */
+    public function getSession(): Session
```

Comment:
I agree, but I couldn't find how.
It's the `protected/vendor/yiisoft/yii2/web/User.php(690)` calling it because the property `enableSession` of this class is `true` even when executing it via command line...

Reply:
@luke- see my answer here: https://github.com/humhub/humhub/issues/6347#issuecomment-1563276738